src/main/java/org/sunbird/org/service/ExtendedOrgServiceImpl.java

```diff
@@ -1079,4 +1081,51 @@ private void fetchStateOrMinistryDetails(Map<String, Object> requestBody) {
 			}
 		}
 	}
+
+	@Override
+	public SBApiResponse updateV2(Map<String, Object> orgRequest, String userToken) {
+		logger.info("ExtendedOrgServiceImpl::updateV2::Starting the update of the organization");
+		SBApiResponse outgoingResponse = ProjectUtil.createDefaultResponse(Constants.API_ORG_EXT_UPDATE);
+		String errMsg = validateRequestFieldsV2(orgRequest, outgoingResponse);
+		if (!StringUtils.isEmpty(errMsg)) return outgoingResponse;
+		OrgHierarchyInfo orgHierarchyInfo = new OrgHierarchyInfo();
+		if(!StringUtils.isBlank(MapUtils.getString(orgRequest, Constants.ORG_NAME))){
+			orgRepository.updateOrgNameBySbOrgId((String) orgRequest.get(Constants.ORG_ID), (String) orgRequest.get(Constants.ORG_NAME));
+			orgHierarchyInfo.setOrgName((String) orgRequest.get(Constants.ORG_NAME));
+		}
+		orgHierarchyInfo.setSbOrgId((String) orgRequest.get(Constants.ORG_ID));
+		Map<String, Object> orgDataUpdateResonse = updateOrgDetailsToDB(userToken, orgHierarchyInfo, orgRequest);
+		if (MapUtils.isEmpty(orgDataUpdateResonse) || !orgDataUpdateResonse.get(Constants.RESPONSE_CODE).equals(Constants.OK)) {
```

Comment:
Never check object attribute against constant... It should be always constant against object attribute
what will happen if orgDataUpdateResponse doesn't have responseCode ??

it should be,
!Constants.OK.equalsIgnoreCase(orgDataUpdateResponse.get(Constants.RESPONSE_CODE))